### ISSUE TITLE
Linux Support

### ISSUE THREAD

**Issue by Davilarek** [juanjp600/B3DDecomp#17]
Builds successfully, but crashes trying to run an exe.
```
Unhandled exception. System.ComponentModel.Win32Exception (2): An error occurred trying to start process 'Blitz3DDisasm.exe' with working directory '/home/davil/AUR/B3DDecomp/bin/Debug/net8.0'. No such file or directory
   at System.Diagnostics.Process.ForkAndExecProcess(ProcessStartInfo startInfo, String resolvedFilename, String[] argv, String[] envp, String cwd, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32& stdinFd, Int32& stdoutFd, Int32& stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
   at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   at Program.Main(String[] args) in /home/davil/AUR/B3DDecomp/Blitz3DDecomp/Program.cs:line 25
```

**Comment by juanjp600** [comment]
Should be fixed in 86036d3

**Comment by Davilarek** [comment]
Yup, thanks
